qa/integration/specs/cli/keystore_spec.rb

```diff
@@ -42,7 +42,11 @@
     end
 
     it "works" do
-      keystore_list = @logstash.run_cmd(['bin/logstash-keystore', 'create'], true, 'LOGSTASH_KEYSTORE_PASS' => 'PaSSWD')
+      env = {'LOGSTASH_KEYSTORE_PASS' => 'PaSSWD'}
+      unless ENV['BUILD_JAVA_HOME']
+        env['JAVA_HOME'] = ENV['BUILD_JAVA_HOME']
+      end
```

Comment:
From what I understand the issue is that we need to make JAVA_HOME equal to BUILD_JAVA_HOME, if it is set (e.g. in the jdk matrix tests). So we should have an `if ` instead of `unless` here:

```suggestion
      if ENV['BUILD_JAVA_HOME']
        env['JAVA_HOME'] = ENV['BUILD_JAVA_HOME']
      end
```

Reply:
Yes you a right, I  was thinking "unless nil?" :-(